app/label.rb

```diff
@@ -0,0 +1,11 @@
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, color, _items, id = Random.rand(1..100_000))
+    @id = id
+    @title = title
+    @color = color
+    @items = []
+  end
```

Comment:
Is there a reason why you added an _items argument if not using it? I might be missing something so please explain to me. If there is no reason, kindly remove it since it is not being utilized.